internal/datastore/postgres/postgres.go

```diff
@@ -74,7 +78,7 @@ func init() {
 var (
 	psql = sq.StatementBuilder.PlaceholderFormat(sq.Dollar)
 
-	getRevision = psql.Select("MAX(id)").From(tableTransaction)
+	getRevision = psql.Select("MAX(xid::text::bigint)").From(tableTransaction)
```

Comment:
Should this use a const name for the col name? I know we didn't use it before, but maybe we should now?